### PR TITLE
pin rtv dependencies

### PR DESCRIPTION
**Pull request by dtrodrigues**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by dtrodrigues** [comment]
Per https://github.com/michael-lazar/rtv/issues/696, the project is no longer maintained.

**Comment by dtrodrigues** [comment]
There have been over 5k downloads in the last year since the project stopped actively being maintained: https://formulae.brew.sh/formula/rtv

**Comment by SMillerDev** [comment]
Can you mark it as `deprecated` since the maintenance stop?

File: Formula/rtv.rb [hunk 1]
```diff
@@ -15,14 +15,62 @@ class Rtv < Formula
     sha256 "a12678c3f7c95c3d45c2e631fda28d1da29fd6768f8dcce235b3599f580e264b" => :high_sierra
   end
 
+  deprecate! :date => "June 2, 2019"
```
Comment (sjackman) [inline Formula/rtv.rb:18]: ISO 8601

https://m.xkcd.com/1179/

**Review by sjackman — CHANGES_REQUESTED** [review]

File: Formula/rtv.rb (hunk 1 above)
Comment (dtrodrigues) [reply Formula/rtv.rb:18]: All  of the other deprecation statements within Homebrew are in "Month DD, YYYY" format. Should they be changed as well? The ambiguity referred to in the comic primarily comes when the month is not spelled out and/or the year is represented by 2 digits, but there isn't a different interpretation for the format currently used. I can make the change if that's what's desired though.

```
 ~/homebrew-core/Formula (master) $ grep deprecate *rb  | grep date
elasticsearch@2.4.rb:  deprecate! :date => "February 28, 2018"
elasticsearch@5.6.rb:  deprecate! :date => "March 11, 2019"
go@1.10.rb:  deprecate! :date => "February 25, 2019"
go@1.11.rb:  deprecate! :date => "September 3, 2019"
go@1.12.rb:  deprecate! :date => "February 25, 2020"
go@1.9.rb:  deprecate! :date => "August 4, 2018"
gradio.rb:  deprecate! :date => "November 16, 2019"
kibana@5.6.rb:  deprecate! :date => "March 11, 2019"
php@7.2.rb:  deprecate! :date => "November 30, 2020"
php@7.3.rb:  deprecate! :date => "December 6, 2021"
postgresql@9.4.rb:  deprecate! :date => "February 13, 2020"
```

Comment (sjackman) [reply Formula/rtv.rb:18]: Please use ISO 8601 for this PR. No need to change the other formulae. I'll open a PR to add a `brew style` Rubocop to enforce ISO 8601 date format, unless you'd like to do that, in which case, have at it!

**Review by SMillerDev — DISMISSED** [review]

**Review by SMillerDev — DISMISSED** [review]

**Review by sjackman — DISMISSED** [review]

**Pull request closed without merging**